api/controllers/UserController.js

```diff
@@ -5,42 +5,54 @@
  * @help        :: See https://sailsjs.com/docs/concepts/actions
  */
 
-
-
 module.exports = {
-
-    create: async function(req, res) {
-        if (req.body == null) {
-            return res.send('null body')
-        } else {
-            const userCreated = await user.create(req.body).fetch();
-            return res.json(userCreated);
-        }
-    },
-    read: async function(req, res) {
-        if (req.params.id != undefined) {
-            const readUser = await user.findOne(req.params.id);
-            return res.json(readUser);
-        } else {
-            return res.send('invalid input');
-        }
-    },
-    update: async function(req, res) {
-        if (req.body == null || req.body.id == undefined || Object.keys(req.body) < 2) {
-            return res.send('invalid input');
-        } else {
-            const userUpdated = await user.update(req.body.id).set(req.body).fetch();
-            return res.json(userUpdated);
-        }
-    },
-    delete: async function(req, res) {
-        if (req.params.id != undefined) {
-            const deletedUser = await user.destroyOne(req.params.id);
-            return res.json(deletedUser);
-        } else {
-            return res.send('invalid input');
-        }
+  create: async function (req, res) {
+    try {
```

Comment:
don't use exceptions for input validation.  check this post for more info https://stackoverflow.com/questions/1504302/is-it-a-good-or-bad-idea-throwing-exceptions-when-validating-data#:~:text=The%20performance%20of%20throwing%20an,to%20find%20the%20next%20failure).